sci-libs/cantera/cantera-2.4.0-r2.ebuild

```diff
@@ -0,0 +1,126 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+FORTRAN_NEEDED=fortran
+FORTRAN_STANDARD=90
+
+inherit desktop fortran-2 python-single-r1 scons-utils toolchain-funcs
+
+DESCRIPTION="Object-oriented tool suite for chemical kinetics, thermodynamics, and transport"
+HOMEPAGE="http://www.cantera.org"
+SRC_URI="https://github.com/Cantera/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+cti fortran pch +python test"
+
+REQUIRED_USE="
+	python? ( cti )
+	${PYTHON_REQUIRED_USE}
+	"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	python? (
+		dev-python/numpy[${PYTHON_USEDEP}]
+	)
+	<sci-libs/sundials-5.0.0:0=
+"
+
+DEPEND="
+	${RDEPEND}
+	dev-cpp/eigen
```

Comment:
This is slotted.

Reply:
Currently there is only one slot of eigen in the portage tree so I don't know will the API of used functions be changed in future slots. Slot 2 of eigen was droped before cantera package appeared in portage tree.

Reply:
To be sure it doesn't and to help portage, you should add it though.

Reply:
Do you mean the addition of restriction by specific SLOT (dev-cpp/eigen:3) here?

Reply:
or `:*` if you **know** it won't break.